deployments/helm-chart/values.yaml

```
@@ -151,7 +151,7 @@ controller:
   affinity: {}
 
   ## The topology spread constraints of the Ingress controller pods.
-  # topologySpreadConstraints: {}
+  #topologySpreadConstraints: {}
```

```suggestion
  # topologySpreadConstraints: {}
```
This change looks unrelated 🤔